subsys/bluetooth/audio/pacs.c

```diff
@@ -849,7 +1122,11 @@ int bt_pacs_cap_unregister(enum bt_audio_dir dir, struct bt_pacs_cap *cap)
 		return -ENOENT;
 	}
 
-	pacs_changed(dir);
+#if defined(CONFIG_BT_PAC_SRC_LOC_NOTIFIABLE)
+
+	pacs_set_notify_bit(FLAG_SOURCE_AUDIO_LOCATIONS_CHANGED);
+	bt_conn_foreach(BT_CONN_TYPE_LE, defer_value_ntf, NULL);
+#endif
```

Comment:
Why are we notifying location here? 

Shouldn't we be notifying the PAC records?

Reply:
Yes, you are absolutely right. This is completely incorrect.